basement_bot/plugins/factoids.py

```diff
@@ -26,7 +26,13 @@ def setup(bot):
     bot.add_command(delete_factoid)
 
 
-@commands.command(name="r")
+@commands.command(
+    name="r",
+    brief="Creates custom trigger with a specified output",
+    description="Creates a custom trigger with a specified name that outputs any specified text, including mentions. All triggers are used by sending a message with a '?' appended in front of the trigger name.",
```

Comment:
Is there a way to make this more than one line? I'm surprised the formatter didn't freak out.